src/hooks/index.ts

```diff
@@ -115,3 +115,49 @@ export const useUserFollowing = () : UserFollowing => {
 
     return { following, onFollow, onUnfollow };
 };
+
+type EventsFilter = {
+    later: ZetkinEvent[] | undefined;
+    today: ZetkinEvent[] | undefined;
+    tomorrow: ZetkinEvent[] | undefined;
+    week: ZetkinEvent[] | undefined;
+}
+
+export const useEventsFilter = (events : ZetkinEvent[] | undefined) : EventsFilter => {
+    function sliceString(string : string) {
```

Comment:
The name of this function applies some sort of general slicing operation. What you're actually doing is a very specific slicing operation, and one that is designed specifically to handle dates. I think that should be reflected in the name of the function, e.g. something like `extractISODate()` or something like that.